thorlcr/msort/tsorts.cpp

```diff
@@ -787,6 +790,13 @@ class CThorSorter : public CSimpleInterface, implements IThorSorter, implements
         transferblocksize = TRANSFERBLOCKSIZE;
         isstable = true;
         stopping = false;
+        useTLS = queryMtls();
+#ifdef _CONTAINERIZED
+        traceLevel = getComponentConfigSP()->getPropInt("logging/@detail", 0);
```

Comment:
this is going to be 100 (by default) for most components, is that going to cause excessive tracing?